fineract-provider/src/main/resources/application.properties

```diff
@@ -43,6 +43,11 @@ fineract.mode.batch-manager-enabled=${FINERACT_MODE_BATCH_MANAGER_ENABLED:true}
 fineract.correlation.enabled=${FINERACT_LOGGING_HTTP_CORRELATION_ID_ENABLED:false}
 fineract.correlation.header-name=${FINERACT_LOGGING_HTTP_CORRELATION_ID_HEADER_NAME:X-Correlation-ID}
 
+fineract.partitioned-job.job-properties[0].job-name=LOAN_COB
+fineract.partitioned-job.job-properties[0].chunk-size=${LOAN_COB_CHUNK_SIZE:100}
```

Comment:
Two things:

- can't we shorten the property prefix from `fineract.partitioned-job.job-properties` to just `fineract.partitioned-jobs`; seems a bit redundant to nest the list as the only attribute in yet another nested class (see my comment above)
- we should prefix the environment variables properly to indicate they are introduced by Fineract. Instead of `LOAN_COB_PARTITION_SIZE` I'd suggest `FINERACT_PARTIONED_JOB_PARTITION_SIZE` which is closer to the property name; we had already a cleanup in the past to remove inconsistent variable names, would be great if we could stick with this.

Reply:
for the first thing: you're right, I'm gonna fix that
for the second thing: we're gonna have different partition sizes for different jobs, because there could be a scenario where someone has a huge jobs that take a long time, and they want them to run on 100 workers, and also they have very tiny jobs, that they want to run on 5 workers. That's why I named it after the specific job.